### ISSUE TITLE
Inverted properties for  `readAllOperation` in modbus client

### ISSUE THREAD

**Issue by relu91** [eclipse/thingweb.node-wot#903]
## TL;DR;
Modbus client has a bug when you try to read multiple properties in parallel. Some of these values are swamped and the end result is wrong. Below you can find a longer explanation, I'm assigning this to myself therefore I tried to be precise so that I can spare another additional 2 hours of debugging 🤣 . 

## Issue
The Modbus client has a complex algorithm to compact multiple operations in one single transaction cleverly.  Check the documentation to [read more](https://github.com/eclipse/thingweb.node-wot/tree/master/packages/binding-modbus#implementation-notes). 

### My setup
I have a Modbus device with 3 contiguous input registers encode in a Float32 Little Endian format. In practice:
| Address  | 16-value  |
|---|---|
|  0 |  Most significative 16 of the first value |
|  1 |  Least significative 16 of the first value|
|  2 |  Most significative 16 of the second value |
|  3 |  Least significative 16 of the second value|
|  4 |  .. |
|  .. | ...   |

### The issue
When using the `readAll`, thanks to our non-blocking strategy the Modbus client understands that we are trying to get 3 continuous values and optimize generating one single request. The client correctly returns the buffer but then our binding implementations manipulates the result to obtain a regular array with **big-endian** values in the [execute()](https://github.com/eclipse/thingweb.node-wot/blob/master/packages/binding-modbus/src/modbus-connection.ts#L91) function. However, the logic implemented there does not work for this corner case because it just reverses the all returned buffer:
```ts
 if (
                    this.endianness === ModbusEndianness.LITTLE_ENDIAN_BYTE_SWAP ||
                    this.endianness === ModbusEndianness.LITTLE_ENDIAN
                )
                    result.buffer.reverse();
```

The correct way to translate this little endian 32 Float values to big endian format would be using `result.buffer.swap32()` but I would rather delegate the whole transformation to the codec. 

### Planned fix
I think that payload manipulation should be delegated to codecs as much as possible and protocol-bindings should just hand over the requests/responses. Therefore, I am keen to remove the logic above a bring it to the `octect-codec` and fix it. This implies also fixing the `contentToValue` function that it seems ignores the `byteSeq` parameter

**Comment by relu91** [comment]
> If I understand the situation correctly the _easy_ fix would be to no longer allow for parallel operations. However, I do not know how much performance penalty one would see. Anyhow, I let you decide the right fix 👍 .

Exactly, one naive solution would be to disallow the aggregation of concurrent requests in one single transaction and serialize the messages on Modbus. But I think I would first give it a try to the planned fix above.
